common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java

```diff
@@ -219,4 +228,36 @@ public static String fromByteArray(@NonNull final byte[] bytes) {
     public static byte[] toByteArray(@NonNull final String string) {
         return string.getBytes(ENCODING_UTF8);
     }
+
+    /**
+     * Converts a Date object into a RFC3339 formatted date String
+     *
+     * @param date Date
+     * @return String
+     */
+    @NonNull
+    public static String RFC3339DateToString(@NonNull final Date date) {
+        final String methodName = "RFC3339DateToString";
+        Logger.verbose(TAG + methodName, "RFC3339DateToString is called.");
+        final SimpleDateFormat RFC3339DateFormat = new SimpleDateFormat(RFC3339_DATE_FORMAT, Locale.US);
```

Comment:
Are fully desugared at this point?  Can we move to java.time?

Reply:
I prefer dealing with this later. For now i'll keep this as-is.